dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -531,26 +531,38 @@ git push --set-upstream origin "${branch}"
 
 Create a GitHub issue with the content generated via manual
 execution of the script below. You will use link to that issue in the next step. You need a GITHUB_TOKEN
-set as your environment variable.
-
-You can also pass the token as `--github-token` option in the script.
-You can also pass list of PR to be excluded from the issue with `--excluded-pr-list`.
+set as your environment variable or pass the token as `--github-token` option in the script.
 
 ```shell script
 cd "${AIRFLOW_REPO_ROOT}"
 
-breeze release-management generate-issue-content-providers --only-available-in-dist
+breeze release-management generate-issue-content-providers --only-available-in-dist --github-token TOKEN
 ```
 
-You can also generate the token by following
-[this link](https://github.com/settings/tokens/new?description=Read%20sssues&scopes=repo:status)
+You can generate the token by following
+[this link](https://github.com/settings/tokens/new?description=Read%20issues&scopes=repo:status)
+
+If you see in the output that some of the PRs are just "noise" (i.e. there is no need to verify them
+as they are misc/documentation kind of changes that have no impact on the actual installation of
+the provider or the code of the provider, can optionally pass list of PR to be excluded from
+the issue with `--excluded-pr-list`. This might limit the scope of verification (some providers
+might disappear from the list and list of authors that will be pinged in the generated issue.
```

Comment:
nit: Missing closing braces
```
If you see in the output that some of the PRs are just "noise" (i.e. there is no need to verify them
as they are misc/documentation kind of changes that have no impact on the actual installation of
the provider or the code of the provider, you can optionally pass a list of PRs to be excluded from
the issue with `--excluded-pr-list`). This might limit the scope of verification (some providers
might disappear from the list and the list of authors that will be pinged in the generated issue).
```

Reply:
resolved. Removed the braces altogether.